apps/mdn/interactive-examples/tf/mdninteractive.tf

```diff
@@ -88,7 +88,7 @@ resource "aws_cloudfront_distribution" "s3_distribution" {
 
   enabled             = true
   is_ipv6_enabled     = true
-  comment             = "No comment"
+  comment             = "MDN Interactive-Examples CDN"
```

Comment:
Good catch! Thanks for updating the comments to reflect my changes via the console!